### PR TITLE
fixed more memory leaks in draw tests

### PR DESCRIPTION
**Pull request by sebastianbaginski**
@marcusnaslund review ?

### CODE REVIEW

File: test/draw/gpu/GpuSurfaceTest.ooc [hunk 1]
```diff
@@ -15,16 +15,16 @@ use opengl
...
 		this add("draw red quadrant scale 1:1", func {
 			correctImage := RasterRgba open("test/draw/gpu/correct/quadrant_red.png")
 			gpuImage := gpuContext createRgba(sourceSize)
 			gpuImage canvas fill(ColorRgba transparent)
 			quadrantRed := FloatBox2D new(0.0f, 0.0f, 0.5f, 0.5f)
-			DrawState new(gpuImage) setInputImage(sourceImage) setSourceNormalized(quadrantRed) setDestinationNormalized(quadrantRed) draw()
+			DrawState new(gpuImage) setInputImage(this sourceImage) setSourceNormalized(quadrantRed) setDestinationNormalized(quadrantRed) draw()
 			rasterFromGpu := gpuImage toRaster()
 			expect(rasterFromGpu distance(correctImage), is equal to(0.0f))
 			rasterFromGpu free(); correctImage free(); gpuImage free()
```
Comment (davidpiuva) [inline test/draw/gpu/GpuSurfaceTest.ooc]: We can use the touple free now.

Comment (sebastianbaginski) [inline test/draw/gpu/GpuSurfaceTest.ooc]: I think @marcusnaslund has a PR that adapts the kean code

Comment (marcusnaslund) [inline test/draw/gpu/GpuSurfaceTest.ooc]: It's already in, so presumably you have not rebased?

Comment (sebastianbaginski) [inline test/draw/gpu/GpuSurfaceTest.ooc]: Do I have to do that now ? It will restart the tests and there are no conflicts

Comment (marcusnaslund) [inline test/draw/gpu/GpuSurfaceTest.ooc]: Not at all.

**Pull request merged**